cmd/node/factory/structs.go

```diff
@@ -595,6 +601,17 @@ func startResetingFloodPreventer(floodPreventer p2p.FloodPreventer) {
 	}()
 }
 
+func handleTopicFloodPreventer(topicFloodPreventer p2p.TopicFloodPreventer, config *config.TopicAntifloodConfig) {
+	topicFloodPreventer.SetMaxMessagesForTopic("heartbeat", config.HeartbeatMaxMessagesPerSec)
+	go func() {
```

Comment:
this can be merged with the startResettingFloodPreventer, init can be taken out of this function

Reply:
done